service/controller/v3/resource/legacy/backoff.go

```diff
@@ -0,0 +1,32 @@
+package legacy
+
+import (
+	"time"
+
+	"github.com/cenkalti/backoff"
+)
+
+const (
+	LongMaxWait  = 30 * time.Minute
+	ShortMaxWait = 2 * time.Minute
+)
+
+const (
+	LongMaxInterval  = 60 * time.Second
+	ShortMaxInterval = 5 * time.Second
```

Comment:
Are Long versions used?

Reply:
Here not right now. This is actually copy/paste from e2e-harness where we use both. I do not mind though. At some point we may want to put this into a common place. Uncertain.